website/blog/2023-01-03-resilient-relay-apps.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: Resilient Relay Applications
+author: Ernie Turner
+description: Resilient Relay Applications
+hide_table_of_contents: false
+---
+
+import Image from '@site/components/Image';
+
+:::tip Guest Post
+
+This is a guest post written by Ernie Turner, a Staff Engineer at Coinbase. Coinbase has thoroughly adopted Relay in their applications and is a strong ally of the Relay Team. Last year they helped co-develop the [Relay VSCode extension](https://marketplace.visualstudio.com/items?itemName=meta.relay). Ernie has agreed to share this internal enginnering blog post with us.
+
+:::
```

Comment:
I wasn't really sure what to write here, so definitely give me some feedback on this. I'm open to changing anything here.